infrastructure/development/variables.tf

```diff
@@ -0,0 +1,313 @@
+  default     = "https://github.com/teamdigitale/digital-citizenship-functions"
+  description = "The GitHub repository that must be associated to the function app"
+}
+
+variable "azurerm_functionapp_git_branch" {
+  default     = "funcpack-release-latest"
+  description = "The branch of the GitHub repository that must be associated to the function app"
+}
+
+variable "azurerm_storage_queue_emailnotifications" {
+  type        = "string"
+  description = "Name of the storage queue for email notifications"
+}
+
+variable "azurerm_storage_queue_webhooknotifications" {
+  type        = "string"
+  description = "Name of the storage queue for notifications sent to webhook"
+}
+
+variable "azurerm_storage_queue_createdmessages" {
+  type        = "string"
+  description = "Name of the storage queue for created messages"
+}
+
+variable "azurerm_storage_queue_profileevents" {
+  type        = "string"
+  description = "Name of the storage queue for profile events (create / update)"
+}
+
+variable "azurerm_cosmosdb_collections" {
+  type        = "map"
+  description = "Name and partition keys of collections that must exist in the CosmosDB database"
+}
+
+variable "app_service_portal_git_repo" {
+  type        = "string"
+  description = "URL of the GitHub repository providing the source of the App Service Portal"
+}
+
+variable "app_service_portal_git_branch" {
+  default     = "master"
+  description = "Branch of the GitHub repository providing the source of the App Service Portal"
+}
+
+variable "app_service_portal_post_login_url" {
+  type        = "string"
+  description = "Redirect to this page after developer portal login"
+}
+
+variable "app_service_portal_post_logout_url" {
+  type        = "string"
+  description = "Redirect to this page after developer portal logout"
+}
+
+variable "azurerm_apim_sku" {
+  type        = "string"
+  description = "SKU (tier) of the API management"
+}
+
+variable "azurerm_adb2c_policy" {
+  type        = "string"
+  description = "Name of ADB2C policy used in the API management portal authentication flow"
+}
+
+# TF_VAR_ADB2C_TENANT_ID
+variable "ADB2C_TENANT_ID" {
+  type        = "string"
+  description = "Name of the Active Directory B2C tenant used in the API management portal authentication flow"
+}
+
+# TF_VAR_DEV_PORTAL_CLIENT_ID
+variable "DEV_PORTAL_CLIENT_ID" {
+  type        = "string"
+  description = "Client ID of an application used in the API management portal authentication flow"
+}
+
+# TF_VAR_DEV_PORTAL_CLIENT_SECRET
+variable "DEV_PORTAL_CLIENT_SECRET" {
+  type        = "string"
+  description = "Client secret of the application used in the API management portal authentication flow"
+}
+
+# TF_VAR_DEV_PORTAL_EXT_CLIENT_ID
+variable "DEV_PORTAL_EXT_CLIENT_ID" {
+  type        = "string"
+  description = "Client ID of an application used by the digital citizenship onboarding procedure"
+}
+
+# TF_VAR_DEV_PORTAL_EXT_CLIENT_SECRET
+variable "DEV_PORTAL_EXT_CLIENT_SECRET" {
```

Comment:
last year we were thinking about storing these keys in a remote terraform state to avoid keeping them on our laptops, other options:

* store them in a remote vault (e.g. azure HSM)
* store them in an encrypted form in the public repo 

what do you think about that?